app/js/checkAnswers.js

```diff
@@ -129,3 +123,56 @@ function displayResult(earnedPoints, earnedPercentage, answeredQuestions) {
     document.querySelector(".answered_questions").innerHTML = answeredQuestions
     document.querySelector(".score_percentage").innerHTML = earnedPercentage
 }
+
+
+function questionAnswered() {
+    let answers = getUserAnswers(30)
+    let answersArr = Object.values(answers)
+    let unanswered = []
+    answersArr.forEach(function (value, qid) {
+        qid++
+        if(value == 'unanswered') {
+            unanswered.push(qid)
+        }
+    })
+    if (unanswered.length == 0) {
+        unanswered = false
```

Comment:
You could just return false. No need to set unanswered to false then return unanswered (just return false).

Reply:
thanks, tis old code